identity-appsupport/src/commonMain/kotlin/com/android/identity/appsupport/ui/PassphraseEntryField.kt

```diff
@@ -61,26 +65,33 @@ import org.jetbrains.compose.resources.getString
  *      provided or is [null] (default case), then it uses [ImeAction.Done].
  *      Note: only the [ImeAction.Done] is recognized to have a task that is executed when the
  *      user taps on the keyboard's action button.
+ * @param disableKeyboard optionally provide [true] to prevent requesting focus on [BasicTextField]
+ *      and dismiss all taps on the text field.
+ * @param setText optionally provide [String] to set as the current value of the text field.
  * @param onChanged called when the user is entering text or pressing the "Done" IME action
  */
 @Composable
 fun PassphraseEntryField(
     constraints: PassphraseConstraints,
-    checkWeakPassphrase: Boolean,
+    checkWeakPassphrase: Boolean = false,
     imeAction: ImeAction? = null,
+    disableKeyboard: Boolean = false,
+    setText: String? = null,
```

Comment:
I think `initialText` would be more accurate.

Reply:
I would agree if `setText` was just set initially however it's always being set to a new string whenever a keypad button is tapped